osu.Game/Screens/Select/SongSelect.cs

```diff
@@ -124,9 +125,27 @@ public abstract partial class SongSelect : ScreenWithBeatmapBackground, IKeyBind
         [Resolved]
         internal IOverlayManager? OverlayManager { get; private set; }
 
+        private Bindable<float> backgroundBlurLevel { get; set; } = new BindableFloat();
+
+        private void applyBackgroundBlur(float v)
```

Comment:
dont abbreviate names in method values it just makes things more confusing for no gain

Reply:
gonna fix that right now sorry, bad habits..